internal/connect/command_artifact_delete.go

```diff
@@ -0,0 +1,86 @@
+package connect
+
+import (
+	"fmt"
+
+	"github.com/spf13/cobra"
+
+	pcmd "github.com/confluentinc/cli/v4/pkg/cmd"
+	"github.com/confluentinc/cli/v4/pkg/deletion"
+	"github.com/confluentinc/cli/v4/pkg/examples"
+	"github.com/confluentinc/cli/v4/pkg/resource"
+)
+
+func (c *artifactCommand) newDeleteCommand() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "delete <id-1> [id-2] ... [id-n]",
+		Short: "Delete one or more Connect artifacts.",
+		Args:  cobra.MinimumNArgs(1),
+		RunE:  c.delete,
+		Example: examples.BuildExampleString(
+			examples.Example{
+				Text: "Delete Connect artifact.",
+				Code: "confluent connect artifact delete --cloud aws --environment env-abc123 cfa-abc123",
```

Comment:
```suggestion
				Code: "confluent connect artifact delete cfa-abc123 --cloud aws --environment env-abc123",
```

Nitpick: Cobra definitely allows this, but I think our general convention in the code examples is to put the arguments in front of the flags